weaviate/connect/connection.py

```diff
@@ -40,16 +39,9 @@
 )
 from weaviate.warnings import _Warnings
 
-try:
-    import grpc  # type: ignore
-    from grpc import Channel
-    from weaviate.proto.v1 import weaviate_pb2_grpc
-
-    has_grpc = True
-
-except ImportError:
-    has_grpc = False
```

Comment:
why remove this? I think this is needed for v3

Reply:
Because the gRPC dependencies aren't optional in the v4 client so those imports will always be installed

If that's an issue for usage of the v3 client, I'm just remembering that it has some under-the-hood gRPC stuff if it's installed, then we should change them back to optional